Automattic/wp-calypso
# Components: Remove global input[type='password'] styles

#### Changes proposed in this Pull Request

* Remove global form `input[type='password']` styles in favor of more specific component-level styling. See #45259.

#### Testing instructions

* Test the following against production and ensure that they match stylistically and functionally:
  * `/devdocs/design/form-fields` - the form password example
  * `/log-in?from=woocommerce-onboarding` - the password field after inputting a username
* Verify all `<input type="password" />` instances use either `<FormPasswordInput />` or Woo's `<TextControl />` (disregard Gutenboarding - it uses `@wordpress/components` and has its own sets of resets).
